Patterns/Base-PhoneNumbers.yaml

```diff
@@ -35,6 +35,9 @@ ITPhoneNumberRegex: !paramsRegex
 NLPhoneNumberRegex: !paramsRegex
   def: ((((\(\s?(\+\s?|00)31\s?\)\s?)|(((?<!\d)\+\s?|{WordBoundariesRegex}00)31\s?))?((({WordBoundariesRegex}|(?<=31))0?\d{1,3}|\(\s?0?\d{1,3}\s?\)|\(0\)[-\s]?\d{1,3})((-?[\d]{5,11})|(\s[\d\s]{5,11}))\d))|\b\d{10,12})(?!-){EndWordBoundariesRegex}
   params: [ WordBoundariesRegex, EndWordBoundariesRegex  ]
+SEPhoneNumberRegex: !paramsRegex
+  def: ({WordBoundariesRegex}((\+|00)46\s\d{1,3}\s\d{2,3}\s\d{2,3}(\s\d{2})?{EndWordBoundariesRegex}|\d{2,4}-\d{2,3}\s\d{2,3}(\s\d{2})?){EndWordBoundariesRegex})
```

Comment:
Can you describe in the PR description some of the key cases not previously captured, but nor captured by this beyond just numbers with the country code for Sweden?